nninit.lua

```diff
@@ -270,4 +270,94 @@ nninit.sparse = function(module, tensor, sparsity)
   return module
 end
 
+--[[
+--  Aghajanyan, A. (2017)
+--  Convolution Aware Initialization
+--  arXiv preprint arXiv:1702.06295
+--]]
+nninit.convolutionAware = function(module, tensor, options)
+  -- The author of the paper provided a reference implementation for Keras: https://github.com/farizrahman4u/keras-contrib/pull/60
+
+  -- Make sure that the signal library is available, which provides the Fourier transform
+  local hasSignal, signal = pcall(require, 'signal')
```

Comment:
Sorry, I meant lift this to the top, as in line 2 of the file. Then it's only called when `nninit` is loaded, not every time this function is called.